test/helpers/run.js

```diff
@@ -0,0 +1,20 @@
+const execa = require('execa');
+
+let project;
+
+function setProject(_project = undefined) {
```

Comment:
This is a bit odd to have to do, but it's because of line 11. Happy to change if you can think of a better way to do this without having to pass in the project for every run call.